pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -202,18 +200,16 @@ func (r *ReconcileNode) createOrUpdateCephCron(cephCluster cephv1.CephCluster, c
 	return controllerutil.CreateOrUpdate(r.opManagerContext, r.client, cronJob, mutateFunc)
 }
 
-func (r *ReconcileNode) deletev1betaJob(objectMeta metav1.ObjectMeta) error {
+func (r *ReconcileNode) deletev1betaJob(objectMeta metav1.ObjectMeta) {
 	// delete v1beta1 cronJob on an update to v1 job,only if v1 job is not created yet
 	if _, err := r.context.Clientset.BatchV1().CronJobs(objectMeta.Namespace).Get(r.opManagerContext, prunerName, metav1.GetOptions{}); err != nil {
 		if apierrors.IsNotFound(err) {
 			err = r.client.Delete(r.opManagerContext, &v1beta1.CronJob{ObjectMeta: objectMeta})
 			if err != nil && !apierrors.IsNotFound(err) {
-				return errors.Wrapf(err, "failed to delete CronJob v1Beta1 %q", prunerName)
+				logger.Debugf("could not delete CronJob v1Beta1 %q. %v", prunerName, err)
 			}
```

Comment:
Do you think it would be better to check the error type returned by k8s when v1Beta1 is no longer supported?

Reply:
We could, but at the same time, we don’t really have a good reason to fail the reconcile. My thought was to just log the error and always go ahead and create the v1 cron job.
If both jobs did end up running, it should just be two jobs doing the same thing to prune crashes from ceph. They would prune the same thing if they ran at the same time, or else have nothing to prune, so I wouldn’t expect an error condition at least.